frontend/app/utils/date-utils.ts

```diff
@@ -73,7 +74,20 @@ export function isValidTimeZone(timeZone: string): boolean {
 }
 
 /**
-<<<<<<< HEAD
+ * @param date - Optional date or timestamp to use. If not provided, the current date and time are used.
+ *               Can be a number (milliseconds since epoch), a string (parsable by `new Date()`), or a Date object.
+ * @returns The date formatted in both English ("MMM dd, yyyy") and French ("dd MMM. yyyy").
+ */
+export function dateToLocalizedText(date?: number | string | Date): { dateEn: string; dateFr: string } {
+  const targetDate = new Date(date ?? Date.now());
+  const tzDate = new TZDate(targetDate);
+  return {
+    dateEn: format(tzDate, 'MMM dd, yyyy', { locale: enUS }),
+    dateFr: format(tzDate, 'dd MMM. yyyy', { locale: fr }),
```

Comment:
Should these be `enCA` and `frCA`?

Reply:
Also, does it make sense to use `Date.toLocaleDateString()` here instead of date-fns?

Reason: the less code we have to ship to the browser, the better. If we don't use date-fns' `format()` function, it will presumably be tree-shaken out of the client bundle.